tests/test_interfaces.py

```diff
@@ -8,57 +8,54 @@
     pytest.mark.device_type('vs')
 ]
 
-def test_interfaces(duthosts, enum_frontend_dut_hostname, tbinfo):
+def test_interfaces(duthosts, enum_frontend_dut_hostname, enum_asic_index):
     """compare the interfaces between observed states and target state"""
 
-    duthost = duthosts[enum_frontend_dut_hostname]
-    host_facts = duthost.setup()['ansible_facts']
-    mg_facts  = duthost.get_extended_minigraph_facts(tbinfo)
-
+    duthost = duthosts[enum_frontend_dut_hostname] 
+    namespace = duthost.get_namespace_from_asic_id(enum_asic_index)
+    host_facts = duthost.interface_facts(namespace=namespace)['ansible_facts']['ansible_interface_facts']
+    mg_facts   = duthost.minigraph_facts(host=duthost.hostname, namespace=namespace)['ansible_facts']
```

Comment:
why change from `get_extended_minigraph_facts` to `minigraph_facts` ?

Reply:
Fixed this to use get_extended_minigraph_facts.